tests/Fixtures/Integration/priority/php_unit_expected_exception_annotation_with_root_namespace,no_unused_imports.test

```diff
@@ -0,0 +1,28 @@
+--TEST--
+Integration of fixers: php_unit_expected_exception_annotation_with_root_namespace,no_unused_imports.
+--CONFIG--
+{"php_unit_expected_exception_annotation_with_root_namespace": true, "no_unused_imports": true}
+--EXPECT--
+<?php
+
+class Foo extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @expectedException \Exception
+     */
+    public function testBar()
+    { }
+}
+
+--INPUT--
+<?php
+use Some\Exception;
+
+class Foo extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @expectedException Exception
+     */
+    public function testBar()
+    { }
+}
```

Comment:
please add the prio test to this list/test as well;
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/tests/FixerFactoryTest.php#L193